src/SFA.DAS.EmployerFinance.MessageHandlers/nlog.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
```

Comment:
Can `nlog.xsd` be included in the project to prevent the auto completion errors when viewing this file in an IDE?

Reply:
I can't see the file(s) on the branch?

* https://github.com/SkillsFundingAgency/das-employerfinance/tree/AML-3183_CreateMessageHandlerProject/src/SFA.DAS.EmployerFinance.MessageHandlers
* https://github.com/SkillsFundingAgency/das-employerfinance/tree/AML-3183_CreateMessageHandlerProject/src/SFA.DAS.EmployerFinance.Jobs